mautic/mautic
# Allow hide theme for email and page constructors Mautic 5

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [No]
| New feature/enhancement? (use the a.x branch)      | [Yes]
| Deprecations?                          | [No]
| BC breaks? (use the c.x branch)        | [No]
| Automated tests included?              | [No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Allow Themes to be hidden in the theme-Selection when creating an email or page

#### Idea:
- Have a toggle in the theme’s config.json to hide the theme in the GUI
- So the Mautic GUI should not offer the hidden themes in https://<mymautic>/s/emails/new

#### Steps to test this PR:
- Upload the theme 
- Go to emails/pages
- Create new email/page
- Check if theme is visible in the overview
- Go to CLI and manually change the toggle in the config.json to hide the theme(`cd themes/<name_of_theme> && nano config.json` after add "hide" and set to it true value)
- Go to emails/pages
- Create new email/page or edit already existing
- Check if theme is now hidden


#### Manually add
![image](https://github.com/mautic/mautic/assets/96595532/ff8e9311-5cbe-4990-9dd9-4c120a67407f)

![grafik](https://github.com/mautic/mautic/assets/96595532/d01463c9-285f-495c-aff9-1d56bf069881)
